sops/european-level/GDI-SOP0009_dataset-withdrawal.md

```diff
@@ -0,0 +1,466 @@
+| TBD | To Be Determined |
+| TL | Technical Lead |
+| UP | User portal |
+| VHD | Virtual Helpdesk |
+| WP | Work Package |
+
+| Term          | Definition      |
+|---------------|-----------------|
+|Data controller|The legal owner or organisation that determines the purposes and means of processing the dataset and can authorise its withdrawal|
+|Hard-deletion|Complete, irreversible erasure of the data from all storage media so it cannot be recovered|
+|Requester|The person that initiates the request to the VHD for the dataset to be withdrawn|
+|Soft-deletion|Data is marked as withdrawn and made inaccessible to users but retained internally for audit or limited-term retention|
+
+### 3. Roles and Responsibilities
+See qualifications and responsibilities of the roles at the [**Organisational Roles and Responsibilities**](https://github.com/GenomicDataInfrastructure/standard-operating-procedures/blob/main/docs/GDI-SOP_organisational-roles-and-responsibilities.md) document.
+
+| Role       | Full name       | GDI/node role   | Organisation |
+|------------|-----------------|-----------------|--------------|
+| Author     |Marcos Casado Barbero|Task 4.3 member|EMBL-EBI|
+| Reviewer   |                 |                 |              |
+| Approver   |                 |                 |              |
+| Authorizer |                 |                 |              |
+
+### 4. Purpose
+The purpose of this SOP is to define the process for withdrawing a dataset when requested by its controller European Genomic Data Infrastructure (GDI). Following its procedure ensures the removal is propagated swiftly and consistently across all services while meeting legal, ethical, and audit requirements. With it, GDI nodes protect data-subject rights, prevent continued exposure of withdrawn data, and maintain a transparent, documented record of the action.
+
+### 5. Scope
+This procedure applies to **all datasets within the GDI**, regardless of which system or node they were originally submitted to. It covers datasets submitted via any GDI component, including but not limited to: **Federated EGA** (FEGA), the **GDI Beacon network**, the **GDI User Portal catalog**, and **FAIR Data Portals** (FDP) under GDI (see more details at [Step 7](#87-per-system-dataset-withdrawal)). 
+
+As a European-level SOP, it is designed to be directly implementable by all GDI nodes and services. The SOP addresses both **full dataset withdrawals** (removing an entire dataset from GDI) and **partial withdrawals** (removing or retracting a portion of a dataset, such as specific individuals' data). This SOP focuses on the **overarching process and coordination** required to withdraw the data across the GDI ecosystem.
+
+It encompasses **withdrawals triggered by a requester**. This entity may be the data controller, a data subject (e.g., withdrawing their consent), legal or ethical entity fulfilling their obligations, or any GDI member due to dataset quality issues or technical needs.
```

Comment:
The request should just come from the legal entity that signed the DPA with the GDI node. i.e data subject should contact the data submitter, and then the data submitter requests the withdrawal to the GDI node. Same as legal or ethical entities, they should communicate first with the data controllers and then the data controllers make the request to GDI.